all/pom.xml

```diff
@@ -238,7 +238,7 @@
                         <artifactId>filevault-package-maven-plugin</artifactId>
                         <executions>
                             <execution>
-                                <id>generate-metadata-cloud</id>
+                                <id>default-metadata</id>
```

Comment:
you need to have two executions for goal `metadata`: one for classic called `default-metadata` (without classifier) and one for Cloud (with classifier) with a different execution id.

Reply:
Maybe the logic is not clear to me... Using profile "cloud" will only build the cloud package? So you either build one or the other but never both in one reactor build? I would prefer a solution where I can build both in one run.

Reply:
I would like an approach like in https://github.com/adobe/aem-core-wcm-components/blob/main/all/pom.xml, where profile cloud just builds both artifacts.

Reply:
cool, I didn't know about the exclude selector (~cloud) and that was the reason I didn't get it working to have both compiled at the same time. PR is updated to use this.

I saw that the validate-package-cloud doesn't pick up the correct package to validate if only the classifier is mentioned as configuration, I also had to explicitly mention the packageFile. Not sure if this is a bug?